### PR TITLE
Exposing typography as mixins for more flexible and quicker consumption

### PR DESCRIPTION
**Pull request by dlockhart**
@omsmith @dbatiste @njostonehouse I don't intend to merge this right away, just looking for feeback. Look for a corresponding PR in BSI that consumes this.

Problem: our fonts are loading in way too late in the rendering pipeline, in all browsers. The browser doesn't even know to fetch `fonts.css` from the CDN until after the web component code has initialized and run, then it still waits to download the `WOFF` files until it determines they're needed.

This PR copies most of our core typography code into Sass mixins, that places like BSI can call into directly.

Advantages: more flexible in how you consume typography, more performant, less susceptible to shadow DOM.

Disadvantages: duplicate code, having multiple ways to do things can be confusing

### CODE REVIEW

**Comment by awikkerink** [comment]
Did you do a rebase + merge?

**Comment by awikkerink** [comment]
If you did, try this:

```
git fetch origin
git checkout typography-sass
git rebase origin/master
git push -f origin typography-sass # this line is key. DONT do A git pull first!
```

**Comment by dlockhart** [comment]
K will try that, thanks

**Comment by dlockhart** [comment]
That looks better

File: test/acceptance/typography.test.js [hunk 1]
```diff
@@ -53,4 +54,18 @@ polymerTests(browsers, function(test) {
 		size: '1024x768',
 		tags: ['desktop']
 	});
+
+	test('typography-mixins-mobile', {
+		endpoint: endpoint,
```
Comment (awikkerink) [inline test/acceptance/typography.test.js]: `endpointMixins`

Comment (dlockhart) [inline test/acceptance/typography.test.js]: Fixed

**Comment by awikkerink** [comment]
Are you going to add image comparisons for the dumps?

**Comment by dlockhart** [comment]
I re-ran the dump, yes, and did a local image compare to make sure everything was as expected. I can add some kind of image compare in here but that's going to be more work... I need to get some of this stuff into the LMS if it's going to get any amount testing done this release. Maybe later.

**Pull request merged**